CMakeLists.txt

```diff
@@ -268,7 +268,15 @@ IF((SDL_BACKEND STREQUAL "SDL") AND NOT USE_SDL2)
 ENDIF()
 
 IF((SDL_BACKEND STREQUAL "SDL2") OR USE_SDL2)
-	INCLUDE(FindSDL2)
+	IF(WIN32 AND _VCPKG_INSTALLED_DIR)
+		find_package(SDL2 CONFIG REQUIRED)
+		GET_TARGET_PROPERTY(SDL2_INCLUDE_DIR SDL2::SDL2 INTERFACE_INCLUDE_DIRECTORIES)
```

Comment:
Which version of cmake does this syntax require? We currently set the minimum to 3.1, but I'm not sure that's enough (no reason not to bump it though).

Also, can't this just replace FindSDL2? SDL2 now ships its own and any platform hacks can be done here.

Reply:
I've tried to avoid intrusion into non VCPkg paths, but yes, the CMake modules should probably be looked into.

Reply:
To me it looks like we can just remove it and make the new block the default.

Reply:
Well, I wasn't able to test on other platforms, so I dared not.

Reply:
We have buildbots for that, but also, the cmake versopm question remains open. AppVeyor ships a very recent version, so the results are not conclusive.

Reply:
Could it be its own PR? After a more thorough investigation.

Reply:
The cleanup, sure, but it's still important to know if we need to bump the minimal cmake version.

Reply:
Oh, all the the syntax is pre-3.0 as far as I understand.